nik4.py

```diff
@@ -199,6 +211,8 @@ def add_fonts(path):
 	parser.add_argument('--add-layers', help='Map layers to include, comma-separated')
 	parser.add_argument('--hide-layers', help='Map layers to hide, comma-separated')
 
+	parser.add_argument('-P', '--projection', help='EPSG code as EPSG:1234 or Proj4 string', default=EPSG_3857)
```

Comment:
What do you think of omitting `EPSG:` for numeric EPSG projections? E.g. using `-P 1234` in this case?

Reply:
I don't have a strong opinion on that. I used "EPSG:1234" because ogr2ogr uses it. But I am aware that the command line user interface of ogr2ogr is – well – strange. Having "EPSG:" at the beginning makes it easy to decide whether I have to prefix it with `+init=` before I hand it over to Mapnik, or not.

Reply:
To me, it would just look better. Users won't type `espg` by mistake, like I always do, and you would need to do only `.isdigit()` instead of `.lower().startswith()`.

Reply:
ok, fixed